ehrql/tables/beta/tpp.py

```diff
@@ -85,51 +85,67 @@ def for_patient_on(self, date):
 
 @table
 class apcs(EventFrame):
+    """
+    Admitted patient care spells (APCS) data is provided via the NHS Secondary Uses Service.
```

Comment:
```suggestion
    Admitted Patient Care Spells (APCS) data is provided via the NHS Secondary Uses Service.
```

Reply:
I'd looked very quickly and the result I saw hadn't capitalised the abbreviation, but it seems like that the capitalised form is more common.